_observing-your-data/event-analytics.md

```diff
@@ -8,57 +8,87 @@ redirect_from:
 
 # Event analytics
 
-Event analytics in Observability is where you can use [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index) (PPL) queries to build and view different visualizations of your data.
+Event analytics in OpenSearch Observability lets you create and see various data visualizations using [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index) (PPL) queries.
 
 ## Getting started with event analytics
 
-To get started, choose **Observability** in OpenSearch Dashboards and then choose **Event analytics**. If you want to start exploring without adding any of your own data, choose **Add samples**, and Dashboards adds sample visualizations you can interact with.
+To get started, choose **Observability** in OpenSearch Dashboards and then choose **Event analytics**. If you want to start exploring without adding your own data, choose **Add samples**. Dashboards adds sample visualizations you can interact with.
```

Comment:
Is "Event analytics" now "Logs?"

<img width="1188" alt="Screenshot 2024-02-15 at 7 49 36 PM" src="https://github.com/opensearch-project/documentation-website/assets/105296784/46e9c188-a24b-4e7a-9ae0-3832ba2000ff">

Reply:
It is in this page where we talk about the log explorer for the event analytics. This is a feature to help with event analytics as it helps explore the logs.